### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by Obey23** [AtiLion/BotNeck-Bot#8]
I'd love to see a quote and prune/purge feature in this

Quoting will grab the contents of a message with an id?

Prune/purge with options for self and all, so you can delete the last X amount of your messages, or all user messages (would have to check if user has the permission)

**Comment by AtiLion** [comment]
I like the quoting idea, and I will most likely implement it.

However I do have concern of the prune/purge commands. The issue is, this bot is designed to send the smallest amount of requests to Discord as possible, that way Discord doesn't ban it instantly due to API abuse. Prune/Purge would require me to make a ton of requests to Discord which will most likely cause an API rate limit or possibly ban.

So in short, while I will be making the quote system, I will not be attempting to do the prune/purge, I highly recommend you use an existing Discord bot for that.

Thank you for your recommendation

**Comment by AtiLion** [comment]
Quote feature was implemented in the latest version of Discord.

Closing this request.
Thank you for your suggestion
